webapp/src/app/home/home.component.ts

```diff
@@ -13,9 +14,21 @@ import { lastValueFrom } from 'rxjs';
 export class HomeComponent {
   leaderboardService = inject(LeaderboardService);
 
+  // timeframe for leaderboard
+  // example: 2024-09-19T10:15:30+01:00
```

Comment:
Do we need the timestamp? Wouldn't date be enough?

Reply:
Yes, of course. I just used a full example as demonstration. Every string, that is parsable as OffsetDateTime in Java, is valid here.